check_links.py

```diff
@@ -0,0 +1,97 @@
+import os
+import urllib.request
+from bs4 import BeautifulSoup
+import markdown
+import re
+
+
+def get_html():
+    lst_html = []
+    # search path is hard coded with ref of this script locaiton
+    for root, dirs, files in os.walk(os.path.join(".","docs","build","html"), topdown=False):
```

Comment:
docs/build are not on our github repo, sphinx builds it on their own runner that I believe we don't have access to. Thus, instead of checking these html files, we have to make do with the markdowns.

Reply:
ok i will update the check link script to be in the scripts folder and i will remove the html checks and just leave with .rst and .md